src/index.js

```diff
@@ -317,7 +317,18 @@ module.exports.warmUp = async (event, context) => {
   console.log("Warm Up Start");
   
   const invokes = await Promise.all(functions.map(async (func) => {
-    console.log(\`Warming up function: \${func.name} with concurrency: \${func.config.concurrency}\`);
+    let concurrency = func.config.concurrency;
+    const functionConcurrency = process.env["WARMUP_CONCURRENCY_" + func.name.toUpperCase().replace(/-/g, '_')]
+    
+    if (functionConcurrency) {
+      concurrency = parseInt(functionConcurrency);
+      console.log(\`Function environment variable warmup concurrency found: \${concurrency}. Overwrote configured concurrency for \${func.name}\`);
+    } else if (process.env.WARMUP_CONCURRENCY) {
+      concurrency = parseInt(process.env.WARMUP_CONCURRENCY);
+      console.log(\`Global environment variable warmup concurrency found: \${concurrency}. Overwrote configured concurrency for \${func.name}\`);
+    }
+    
+    console.log(\`Warming up function: \${func.name} with concurrency: \${concurrency}\`);
```

Comment:
This log message will be a bit redundant when using the env variables.

I'd rather have a single log like:
```js
    let concurrency;
    const functionConcurrency = process.env["WARMUP_CONCURRENCY_" + func.name.toUpperCase().replace(/-/g, '_')]
    if (functionConcurrency) {
      concurrency = parseInt(functionConcurrency);
      console.log(\`Warming up function: \${func.name} with concurrency: \${concurrency} (from function-specific environment variable)\`);
    } else if (process.env.WARMUP_CONCURRENCY) {
      concurrency = parseInt(process.env.WARMUP_CONCURRENCY);
      console.log(\`Warming up function: \${func.name} with concurrency: \${concurrency} (from global environment variable)\`);
    } else {
      concurrency = func.config.concurrency;
      console.log(\`Warming up function: \${func.name} with concurrency: \${concurrency}\`);
    }
```
so there is only a single line that contains the function name, the concurrency and where the concurrency comes from.